docs/azure/deployment.md

```diff
@@ -17,12 +17,12 @@ Whether your workflow is through the [Azure CLI](https://learn.microsoft.com/cli
 
 These tutorials from Microsoft Learn describe different ways of creating and deploying apps to Azure via Visual Studio Code:
 
-Tutorial | Description
---- | ---
-[Deploy Azure Functions](https://learn.microsoft.com/azure/developer/javascript/tutorial/azure-function-cosmos-db-mongo-api) | Build and manage Azure Functions serverless apps directly in VS Code with the Azure Functions extension.
-[Deploy using the App Service extension](https://learn.microsoft.com/azure/app-service/tutorial-nodejs-mongodb-app) | Manage Azure resources directly in VS Code with the Azure App Service extension.
-[Deploy using Docker](https://learn.microsoft.com/azure/developer/javascript/tutorial/tutorial-vscode-docker-node/tutorial-vscode-docker-node-01) | Deploy your website using a Docker container.
-[Deploy a static website](https://learn.microsoft.com/azure/static-web-apps/getting-started) | Create, deploy, and update a static website.
+Tutorial(s) | Description | Framework / Language
+--- | --- | ---
+[Deploy a static website](https://learn.microsoft.com/azure/static-web-apps/getting-started) | Create, deploy, and update a static website | Angular, Blazor, React, or Vue
+[Deploy Node.js web apps](https://code.visualstudio.com/docs/nodejs/nodejs-deployment) | Deploy web apps, containerized apps, or serverless code | Node.js
```

Comment:
Use relative links to the topic Markdown file rather than absolute links.
This allows people to also navigate the topics in VS Code, on GitHub, and on our staging servers.
The links would be:
/docs/nodejs/nodejs-deployment.md
/docs/python/python-on-azure.md
/docs/java/java-on-azure.md